west.yml

```diff
@@ -195,7 +195,7 @@ manifest:
       groups:
         - debug
     - name: sof
-      revision: 779f28ed465c03899c8a7d4aaf399856f4e51158
+      revision: 9a7a5ce17dcb339e3fd667c8f12dd50db13afe31
```

Comment:
Try adding `submodules: true' and see if it works?

Reply:
do not think this is the problem, the way the module has been updated is wrong, I am looking into it right now.